include/zenoh-pico/api/macros.h

```diff
@@ -279,8 +279,10 @@ template<> inline int8_t z_drop(z_owned_publisher_t* v) { return z_undeclare_pub
 template<> inline void z_drop(z_owned_keyexpr_t* v) { z_keyexpr_drop(v); }
 template<> inline void z_drop(z_owned_config_t* v) { z_config_drop(v); }
 template<> inline void z_drop(z_owned_scouting_config_t* v) { z_scouting_config_drop(v); }
+#if Z_FEATURE_SUBSCRIPTION==1
```

Comment:
This one, I'd say we should either have all the feature token in the file or none.

Did you need this for a reason?

Reply:
I seem to remember that otherwise it didn´t compile with Z_FEATURE_SUBSCRIPTION=0
```
lib/zenoh-pico/include/zenoh-pico/api/macros.h:282:72: error: 'z_undeclare_pull_subscriber' was not declared in this scope
```